### PR TITLE
Error thrown when installing documentation with wrong node version

### PR DESCRIPTION
**Pull request by bidoubiwa**
fixes: #963

Example of output:

```
➜ nvm use 10
Now using node v10.22.1 (npm v6.14.6)
➜ yarn
yarn install v1.22.5
[1/5] 🔍  Validating package.json...
error documentation@1.0.0: The engine "node" is incompatible with this module. Expected version ">=12 <15". Got "10.22.1"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
➜ nvm use 16
Now using node v16.2.0 (npm v7.13.0)
➜ yarn 
yarn install v1.22.5
[1/5] 🔍  Validating package.json...
error documentation@1.0.0: The engine "node" is incompatible with this module. Expected version ">=12 <15". Got "16.2.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
➜ nvm use 14
Now using node v14.17.0 (npm v6.14.13)
➜ yarn 
yarn install v1.22.5
[1/5] 🔍  Validating package.json...
```

Apparently netlify was still using `node 10` without any problem. But since I required in the package json the node version to be between 12 and under 15, the build first failed. 
Per [their documentation](https://docs.netlify.com/configure-builds/manage-dependencies/#node-js-and-javascript) `.nvmrc` acts as the node version netlify should use.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,9 @@
     "vuepress-plugin-sitemap": "^2.3.1",
     "vuepress-plugin-zooming": "^1.1.8"
   },
+  "engines": {
+    "node": ">=12 <15"
```
Comment (guimachiavelli) [inline package.json:49]: I think you mentioned this over slack, but why do we need the `engines` field? If I understand correctly both netlify and local give precedence to .nvmrc, no?

**Review by guimachiavelli — APPROVED** [review]

**Comment by dichotommy** [comment]
Should we also create a `.npmrc` with `engine-strict=true`, or is that no longer necessary?

**Comment by bidoubiwa** [comment]
> Should we also create a .npmrc with engine-strict=true, or is that no longer necessary?

I tried it and it worked without setting this

**Comment by bidoubiwa** [comment]
**Update**

I tried without the `engine` and with only the `.nvmrc` using node 10 and having nvm locally, I didn't stop the `yarn install`

**Pull request merged**